src/cookbooks/bigquery.md

```diff
@@ -190,6 +191,19 @@ You can create views in BigQuery if you have access via [GCP BigQuery Console](b
     - Prefix your view with your username. If your username is `username@mozilla.com` create a table with `username_my_view`.
 - See [Creating Views](https://cloud.google.com/bigquery/docs/views) documentation for detailed steps.
 
+### Accessing map-like fields
+
+BigQuery currently lacks native map support and our workaround is to use a STRUCT type with fields named [key, value]. We've created a user-defined function (UDFs) that provides key-based access with the signature: `udf.get_key(<struct>, <key>)`. The example below generates a count per `reason` key in the `event_map_values` field in the telemetry events table for Normandy unenrollment events from yesterday.
+```sql
+SELECT udf.get_key(event_map_values, 'reason') AS reason,
+       COUNT(*) AS EVENTS
+FROM telemetry.events
+WHERE submission_date_s3 = DATE_SUB(CURRENT_DATE(), INTERVAL 1 DAY)
```

Comment:
```suggestion
WHERE submission_date = DATE_SUB(CURRENT_DATE(), INTERVAL 1 DAY)
```

Both submission_date and submission_date_s3 exist now in the user-facing views. Probably best to encourage submission_date now for future compatibility.